backend/app/services/rag/storage/base.py

```
@@ -108,15 +117,26 @@ def get_index_name(self, knowledge_id: str, **kwargs) -> str:
             self._validate_knowledge_id(knowledge_id, mode)
             prefix = self._validate_prefix(mode)
 
-            # Validate rollingStep
-            step = self.index_strategy.get("rollingStep", 5000)
+            # Validate rollingStep (number of knowledge_ids per index bucket)
+            step = self.index_strategy.get("rollingStep", 10)
             if not isinstance(step, int) or step <= 0:
                 raise ValueError(f"rollingStep must be a positive integer, got: {step}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)